### PR TITLE
FIX: ln invoice expire time

### PR DESCRIPTION
**Pull request by limpbrains**
fixing #3740 fix

If invoice expired. BW will show "Expired in Expired minutes"

Screenshot 2021-09-06 at 11 26 40

- try to avoid construction sentences by joining strings in code. Better to have new loc string for each case.
- if you drastically change loc string in en.json and it now has different meaning, it is better to create new key for it. Otherwise it should be deleted from all other loc files.

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/788dg0hp5u2ezeve38j6d8k86m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8141)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/x94vfy3hyrvfzj112wupm75kf0?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8142)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @marcosrdz] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/marcosrdz

File: loc/es.json [hunk 1]
```diff
@@ -554,14 +568,14 @@
...
-    "coins_selected": "({number}) monedas (coins) seleccionadas",
+    "coins_selected": "({number}) \"coins\" seleccionadas",
     "empty": "Esta cartera no tiene fondos en este momento",
     "freeze": "Congelar",
     "freezeLabel": "Congelar",
     "freezeLabel_un": "Descongelar",
     "header": "Coin control",
-    "use_coin": "Usar moneda (coin)",
-    "use_coins": "Usar monedas",
+    "use_coin": "Usar \"coin\"",
```
Comment (marcosrdz) [inline loc/es.json:577]: Something went wrong here...

**Review by marcosrdz — CHANGES_REQUESTED** [review]

File: loc/es.json (hunk 1 above)
Comment (limpbrains) [reply loc/es.json:577]: I just pulled fresh changes from Transifex. Are they wrong again?

Comment (marcosrdz) [reply loc/es.json:577]: Yup. Ill mark the rest

File: loc/es.json [hunk 2]
```diff
@@ -554,14 +568,14 @@
   },
   "cc": {
     "change": "Cambio",
-    "coins_selected": "({number}) monedas (coins) seleccionadas",
+    "coins_selected": "({number}) \"coins\" seleccionadas",
```
Comment (marcosrdz) [inline loc/es.json:571]: This too

File: loc/es.json [hunk 3]
```diff
@@ -554,14 +568,14 @@
...
     "empty": "Esta cartera no tiene fondos en este momento",
     "freeze": "Congelar",
     "freezeLabel": "Congelar",
     "freezeLabel_un": "Descongelar",
     "header": "Coin control",
-    "use_coin": "Usar moneda (coin)",
-    "use_coins": "Usar monedas",
+    "use_coin": "Usar \"coin\"",
+    "use_coins": "Usar \"coins\"",
```
Comment (marcosrdz) [inline loc/es.json:578]: same

**Review by marcosrdz — CHANGES_REQUESTED** [review]

**Comment by limpbrains** [comment]
@marcosrdz reverted

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/n3vvgrnrj1vwqb9jwvpzjhjenr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8151)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**